.claude/skills/extract-ecore.md

```
@@ -0,0 +1,176 @@
+4. **Output Formats**:
+
+   **Package Overview**:
+   ```
+   | Package | Classes | Description |
+   |---------|---------|-------------|
+   | namespace | 8 | Named elements, packages |
+   | structure | 25 | Classes, attributes, relations |
+   ```
+
+   **Class Details**:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)